### ISSUE TITLE
Grade of separation

### ISSUE THREAD

**Issue by gabfus** [albar965/littlenavmap#970]
Hello, you can add on information from IA Flight a new option  to see grade of separation from my plane (es. +40 or -35 with arrow up or down to show if plane is climb or descent) how is showed in Garmin G1000 Maps?
See image
![Immagine 2022-12-27 170028](https://user-images.githubusercontent.com/1875431/209696427-87107601-1c33-41e1-a5ac-5d38b01b2119.png)

Thanks for your job.

**Comment by albar965** [comment]
Like the TCAS display? 
Makes sense. I keep the idea for an update but cannot promise if it will be in the next one.

**Comment by gabfus** [comment]
Yes, I use cassna with g1000 but I think the symbols are same the Boeing tcas.

Thanks
